src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/PublicAPI.Shipped.txt

```diff
@@ -217,6 +217,8 @@ Microsoft.VisualStudio.ProjectSystem.VS.Properties.AbstractProjectConfigurationP
 Microsoft.VisualStudio.ProjectSystem.VS.Properties.AbstractProjectConfigurationProperties.PlatformTarget.set -> void
 Microsoft.VisualStudio.ProjectSystem.VS.Properties.AbstractProjectConfigurationProperties.Prefer32Bit.get -> bool
 Microsoft.VisualStudio.ProjectSystem.VS.Properties.AbstractProjectConfigurationProperties.Prefer32Bit.set -> void
+Microsoft.VisualStudio.ProjectSystem.VS.Properties.AbstractProjectConfigurationProperties.PreferNativeArm64.get -> bool
```

Comment:
And if we don't need a new public member on `AbstractProjectConfigurationProperties` we don't need these lines, either.

Reply:
removed!